multicorn_das/__init__.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, fdw_options, fdw_columns):
         log_to_postgres(f'Initializing DASFdw with options: {fdw_options}', DEBUG)
         self.url = fdw_options['das_url']
         log_to_postgres(f'Connecting to {self.url}', DEBUG)
-        self.channel = grpc.insecure_channel(self.url)
+        self.__create_channel()
```

Comment:
There are cases where in the postgres shell, FDW would never reconnect because it was using a broken channel. Worked when re-opening it.